web/e2e/page-objects/entities/entities.ts

```diff
@@ -34,11 +34,14 @@ export class EntityPage extends AppPage {
   }
 
   async clickEditEntity(entityName: string) {
-    await browser.executeScript(`window.document.getElementById("aeb-${entityName}").getElementsByClassName("edit-start")[0].click()`);
+    let e = element(by.css(`#aeb-${entityName} .fa-pencil`));
+    await browser.executeScript("arguments[0].click();", e)
   }
 
-  deleteEntityButton(entityName: string) {
-    return element(by.css('svg > .nodes * #fo-' + entityName + ' > .foreign > app-entity-box > .entity-def-box > app-resizable > .title > .edit-area > .delete-entity > i'));
+  async deleteEntity(entityName: string) {
```

Comment:
name this clickEditEntity since it's a clicking action

Reply:
changed.